openfisca_core/columns.py

```diff
@@ -32,6 +37,7 @@ class Column(object):
     formula_class = None
     is_period_size_independent = False  # When True, value of column doesn't depend from size of period (example: age)
     is_permanent = False  # When True, value of column doesn't depend from time (example: ID, birth)
+    period_behavior = None
```

Comment:
Can it **only** be called for that period, or is it its **definition** period but the engine will do its best to fit whatever you request from it?